Directory.Build.props

```diff
@@ -1,27 +1,32 @@
 <Project>
-  <PropertyGroup Label="Package information">
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/IntelliTect/Multitool</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/IntelliTect/Multitool</RepositoryUrl>
-    <Authors>IntelliTect</Authors>
-    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+	<PropertyGroup Label="Package information">
```

Comment:
Isn't some of this duplicating what is in the Directory.Build.props files?

Reply:
This is the directory.Build.props file, just got reformatted

Reply:
Ah opps, sorry I missed that.